src/FunctionApps/python/IntakePipeline/fhir.py

```diff
@@ -37,4 +42,158 @@ def read_fhir_bundles(container_url: str, container_prefix: str) -> Iterator[dic
 
 
 def upload_bundle_to_fhir_server(bundle: dict) -> None:
-    pass
+    """Upload a FHIR bundle to FHIR server"""
+    process_fhir_resource(bundle)
+
+
+def process_fhir_resource(
```

Comment:
Can we get rid of this and call `import_to_fhir(bundle)` directly?

Reply:
Done.